main.py

```diff
@@ -1,32 +1,34 @@
+import asyncio
 import re
+import sys
+from decouple import config
 from typing import Final
-from telegram import Update
-from telegram import Bot
+from telegram import Update, Bot
 from telegram.ext import (
+    Application,
     CommandHandler,
     MessageHandler,
-    Filters,
-    Updater,
+    filters,
     CallbackContext,
 )
-from decouple import config
-import sys
 
-from database import (
```

Comment:
Remember, we still need to have a common Atlas cluster to allow this functions to work.

Reply:
Yeah my bad about that, I will pass you later the connection string for the cluster